### DISCUSSION TITLE
vim bindings in kode

### DISCUSSION THREAD

**Discussion by tobbegd** [Kode/Kha#1334] [Q&A]
total beginner here. It seems you cant install extensions in kode editor? I would like to install the vim bindings. Hiw can l do that? Thanks

**Comment by tcdude** [comment]
Currently it is better to use vscode with the `Kha Extension Pack` instead of KodeStudio, since KodeStudio hasn't been updated and there were some major changes since it last got updated. 

This may change when @RobDangerous updates KodeStudio in the future, but for now vscode gives you the best experience for Kha development.

**Reply by tobbegd** [reply]
ok thank you!
